genesyscloud/dependent_consumers/dependent_consumers.go

```diff
@@ -0,0 +1,43 @@
+package dependent_consumers
+
+var dependentConsumerMap map[string]string
+
+func SetDependentObjectMaps() map[string]string {
+	if len(dependentConsumerMap) < 1 {
+		dependentConsumerMap = make(map[string]string)
```

Comment:
Can you please sort this list by the map keys?

Reply:
done